src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -1574,7 +1717,7 @@ public HashSet<(uint256, DateTimeOffset)> RemoveTransactionsByIdsLocked(string w
 
             lock (this.lockObject)
             {
-                IEnumerable<HdAccount> accounts = wallet.GetAccounts();
+                IEnumerable<HdAccount> accounts = wallet.GetAccounts(a => true);
```

Comment:
What does this do?

Reply:
It filters out some of the accounts, in this case it will now return all of them. I think it was actually a bug that it wasn't being used previously, as it was leaving out the cold staking accounts.

I can't see a reason why you'd want to not search the cold staking accounts when removing by ID.